account_info_query.go

```diff
@@ -217,7 +217,7 @@ func (query *AccountInfoQuery) Execute(client *Client) (AccountInfo, error) {
 		cost = query.queryPayment
 	} else {
 		if query.maxQueryPayment.tinybar == 0 {
-			cost = client.maxQueryPayment
+			cost = client.defaultMaxQueryPayment
```

Comment:
<!-- Suggestions are specific requests to improve the subject under review. It is assumed that we all want to do what's best, so these comments are never dismissed as “mere suggestions”, but are taken seriously.  -->
**suggestion:** ‏We can use the newly introduced `client.GetDefaultMaxQueryPayment()` method instead of accessing the unexported field directly.